src/java/test/org/apache/zookeeper/server/ZooKeeperServerMainTest.java

```diff
@@ -64,10 +65,15 @@
 
         public MainThread(int clientPort, boolean preCreateDirs, String configs)
                 throws IOException {
-            this(clientPort, preCreateDirs, ClientBase.createTmpDir(), configs);
+            this(clientPort, preCreateDirs, ClientBase.createTmpDir(), configs, -1);
         }
 
-        public MainThread(int clientPort, boolean preCreateDirs, File tmpDir, String configs)
+        public MainThread(int clientPort, boolean preCreateDirs, String configs, int threshold)
```

Comment:
the `threshold` word in method signature doesn't say much so it requires digging the code to see what it's all about. It would be nice to rename to something closer to its intent (there are plenty of names in this patch already).